htdocs/mrp/mo_production.php

```diff
@@ -962,7 +967,17 @@
 		print '<div class="fichehalfright">';
 		print '<div class="clearboth"></div>';
 
-		print load_fiche_titre($langs->trans('Production'), '', '');
+		$nblinetoproduce = 0;
+		foreach ($object->lines as $line) {
+			if ($line->role == 'toproduce') {
+				$nblinetoproduce++;
+			}
+		}
+		$newlinetext = '';
+		if ($object->status != $object::STATUS_PRODUCED && $object->status != $object::STATUS_CANCELED && $action != 'consumeorproduce' && $action != 'consumeandproduceall') {
+			$newlinetext = '<a href="'.$_SERVER["PHP_SELF"].'?id='.$object->id.'&action=addproduceline">'.$langs->trans("AddNewProduceLines").'</a>';
```

Comment:
I think you forgot the test 
`if ($nblinetoproduce == 0) {`
here. So the link to add a product to product is suggested only if section is empty (no need for the moment if already filled)
Am i right ?

Reply:
Ye but the problem with this is that in Disassemble mode we can only add 1 line of production maybe 
``` 
if ($nblinetoproduce == 0 || $object->mrptype == 1) {
```
could be a good fix which permit to have more than one production in Disassemble mode and only 1 production in Manufacturing

Reply:
Correct. Go this way.

Reply:
Hello
Thanks it's works
Therefore in tab mrp/mo_card.php,  we already enter the product to produce.
So why in mrp/mo_production.php, we must must add this product.
It will be better to transfert the product to produce from mo_card to mo_production.php.
Stef